tesla_ble_mqtt/config.yaml

```diff
@@ -17,33 +17,25 @@ map:
   - share:rw
 startup: services
 options:
-  vin1: ""
-  vin2: ""
-  vin3: ""
-  presence_detection: false
-  ble_mac1: ""
-  ble_mac2: ""
-  ble_mac3: ""
+  vin_list: ""
+  ble_mac_list: ""
+  presence_detection_ttl: 180
   debug: false
```

Comment:
Don't put the false, this will make the setting to show up only in the "additional section".  If someone really wants debug, he'll find it in the "additional section".